.circleci/config.yml

```diff
@@ -99,6 +103,26 @@ jobs:
       - run:
           name: run tests
           command: cd ../integration-tests && export testNode=true && npm run test
+  deploy-beta:
```

Comment:
Same as `deploy` except the last line

Reply:
Both deploy to beta and regular deploy will run when merging to master, that's not what you want to happen, right?

Reply:
The `deploy` should run only on trigger

Reply:
Yes, but the flow for triggering later deploy is still not done.